nextcord/channel.py

```diff
@@ -742,6 +743,9 @@ async def create_thread(
             The type of thread to create. If a ``message`` is passed then this parameter
             is ignored, as a thread created with a message is always a public thread.
             By default this creates a private thread if this is ``None``.
+        invitable: :class:`bool`
+            Whether non-moderators can add other non-moderators to this thread.
+            Only available for private threads and without message.
```

Comment:
```suggestion
            Only available for private threads and created without ``message``.
```
It isn't too clear as to what "and without message" means (as seen above).